ports/polyhook2/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "polyhook2",
-  "version-date": "2023-05-16",
+  "version-date": "2023-08-12",
```

```suggestion
  "version-date": "2023-08-11",
```
`REF 71553ebf0d197e73c49365cc3ab2a6aa326b3c37`
This date should preferably be the upstream commit date.